doc/_static/css/landingpage.css

```diff
@@ -0,0 +1,1526 @@
+html {
```

Comment:
As this file is named `doc/_static/css/landingpage.css`, it shouldn't be part of this pull request.